rootfs/ingest/bin/replctl

```diff
@@ -31,21 +31,22 @@ Index table columns using replication service to Qserv
 #  Imports of standard modules --
 # -------------------------------
 import argparse
-from enum import Enum, auto
 import logging
 import os
 import sys
+from enum import Enum
```

Comment:
I think, having this statement in this line violates a requirement of the LSST Style Guide for sorting imports alphabetically based on the name of the imported package. The right order would be:
```
from enum import Enum
import logging
import os
import sys
```

Reply:
:+1: